### PR TITLE
Detect iPad OS13 as mobile device

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
iPadOS 13 has a new user agent and we have (duplicated) code that has not been updated for this raised in #928.

# Solution
Move check for iOS/Android devices to common utils file and use `isIOS` util function in that file that has the fix.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

File: src/components/Capture/Face.js [hunk 1]
```diff
@@ -10,10 +10,9 @@ import withCrossDeviceWhenNoCamera from './withCrossDeviceWhenNoCamera'
 import GenericError from '../GenericError'
 import FallbackButton from '../Button/FallbackButton'
 import CustomFileInput from '../CustomFileInput'
-import { isDesktop, addDeviceRelatedProperties } from '~utils'
+import { isDesktop, addDeviceRelatedProperties, getMobileOSName } from '~utils'
 import { compose } from '~utils/func'
 import { randomId, upperCase } from '~utils/string'
-import { getMobileOSName } from '~utils/detectMobileOS'
```
Comment (markorodic) [inline src/components/Capture/Face.js:16]: Are we deleting this function in utils/detectMobileOS also?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-970-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-970-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Capture/Face.js (hunk 1 above)
Comment (kopijunkie) [reply src/components/Capture/Face.js:16]: It's been moved to `utils/index.js` to keep all similar utils in the same place so `utils/detectMobileOS` file is now no more.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-970-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://970-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-970-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**